src/importexport/midi/internal/midiexport/exportmidi.cpp

```diff
@@ -220,6 +220,7 @@ void ExportMidi::writeHeader()
 
 bool ExportMidi::write(QIODevice* device, bool midiExpandRepeats, bool exportRPNs, const SynthesizerState& synthState)
 {
+    UNUSED(midiExpandRepeats);
```

Comment:
will be brought back in next PR's, but for now ok

Reply:
I hoped so, this warning the me looked like an error 😜